addons/block_code/ui/blocks/parameter_block/parameter_block.tscn

```diff
@@ -1,19 +1,8 @@
-[gd_scene load_steps=4 format=3 uid="uid://clipm2dd28jde"]
+[gd_scene load_steps=3 format=3 uid="uid://clipm2dd28jde"]
 
 [ext_resource type="Script" path="res://addons/block_code/ui/blocks/parameter_block/parameter_block.gd" id="1_0hajy"]
 [ext_resource type="PackedScene" uid="uid://c7puyxpqcq6xo" path="res://addons/block_code/ui/blocks/utilities/drag_drop_area/drag_drop_area.tscn" id="2_gy5co"]
 
-[sub_resource type="StyleBoxFlat" id="StyleBoxFlat_dbera"]
-bg_color = Color(1, 1, 1, 1)
```

Comment:
Wait were these StyleBoxes removed manually? They are meant to be saved to the scene, somehow I missed this.

Reply:
Hmm I think I reset them to defaults in the editor on the basis that they will be instantiated every time the scene is actually used. I didn't see the square-icon issue... I can investigate tomorrow

Reply:
Yeah, the default stylebox value is actually used as a template, only the color is set on it. Which is kind of weird, maybe we should just create a totally new stylebox at runtime.